DependencyInjection/Configuration.php

```diff
@@ -153,6 +153,11 @@ private function getDbalConnectionsNode()
                     ->useAttributeAsKey('name')
                     ->prototype('scalar')->end()
                 ->end()
+                ->arrayNode('defaultTableOptions')
```

Comment:
you are missing the ``->fixXmlConfig()`` on the parent node for the XML to work properly